### DISCUSSION TITLE
TLDR Viewer Link Support

### DISCUSSION THREAD

**Discussion by max-mykhailenko** [outline/outline#7203] [Ideas]
Could you please add support for TLDR https://www.tldraw.com view only links?
`https://www.tldraw.com/ro/HASH?v=21,-19,1994,952&p=page`

Editor Link looks like `https://www.tldraw.com/r/HASH?v=21,-19,1994,952&p=page`

I suppose that it should be changed in this regex https://github.com/outline/outline/blob/43cf33fc0a8cbe0c4ab38bf4800a3d9a1c65d58d/shared/editor/embeds/index.tsx#L550C11-L550C72

From `/[rsv]/` to something like `/[rsvo]+/` (not strict, but still looks good)

**Comment by tommoor** [comment]
Yeah, please file an issue or PR and it'll get done

**Comment by max-mykhailenko** [comment]
@tommoor Here is it https://github.com/outline/outline/pull/7205 
I'm not familiar with your developing process (linking to issues, tests, etc.) so please do be harsh )
